scripts/globals/job_utils/dancer.lua

```diff
@@ -67,7 +84,62 @@ local function setFinishingMoves(player, numMoves)
     end
 end
 
-xi.job_utils.dancer.stepAbilityCheck = function(player, target, ability)
+local function getStepAnimation(weaponSkillType)
```

Comment:
Not a huge fan of these long conditions for either function, but there's some overlap so I get it.  Given its only 3 conditions that do, I'd almost prefer to table this data

Reply:
Tabled this functions. However, this functions came as they were from weapinskills.lua. The purpose was to take stuff out of that file only.